guides/source/association_basics.md

```diff
@@ -7,20 +7,56 @@ This guide covers the association features of Active Record.
 
 After reading this guide, you will know how to:
 
+* Understand the various types of associations.
 * Declare associations between Active Record models.
-* Understand the various types of Active Record associations.
-* Use the methods added to your models by creating associations.
+* Choose the right association type for your models.
+* Use Single Table Inheritance.
+* Setting up and using Delegated Types.
 
 --------------------------------------------------------------------------------
 
-Why Associations?
+Associations Overview
 -----------------
 
-In Rails, an _association_ is a connection between two Active Record models. Why do we need associations between models? Because they make common operations simpler and easier in your code.
+_Associations_ in Rails are special macro style calls that make it easy to
+connect different models. You can use them to tell Rails how your models relate
+to each other, which helps you manage your data more effectively, and makes
+common operations simpler and easier to read.
 
-For example, consider a simple Rails application that includes a model for authors and a model for books. Each author can have many books.
+INFO: A macro-style call is a method that generates or modifies other methods at
+runtime, allowing for concise and expressive declarations of functionality, such
+as defining model associations in Rails. For example, `has_many :comments`.
 
-Without associations, the model declarations would look like this:
+When you set up an association, you instruct Rails to maintain the [Primary
+Key](https://en.wikipedia.org/wiki/Primary_key) and [Foreign
+Key](https://en.wikipedia.org/wiki/Foreign_key) information between instances of
+the two models, ensuring that your data stays consistent and properly linked.
```

Comment:
Perhaps pedantic but the foreign key and primary key information is maintained by the database that Rails can set up.